docs/building-features/foreman-issues.md

```diff
@@ -1,4 +1,4 @@
-# Foreman Issues
+# Troubleshooting Foreman with React on Rails
```

Comment:
Foreman & ReactOnRails have nothing to do with each other.

This should probably be titled "Troubleshooting issues with foreman, overmind, or other process managers"